### DISCUSSION TITLE
Get VSCode to have the same decorations like Firefox

### DISCUSSION THREAD

**Discussion by KyoukaHeiwa** [JaKooLit/Hyprland-Dots#174] [Q&A]
Hi, I wanted to know how to enable the same features/decorations applied to Firefox (such as transparent background) to the app so it works while in focus?

**Comment by JaKooLit** [comment]
edit 

~/.config/hypr/UserConfigs/WindowRules.conf

start the apps you wanted. ran hyprctl clients through kitty to get proper class name. Note it is upper/lower case sensitive
